lua/orgmode/colors/highlights.lua

```diff
@@ -88,6 +118,7 @@ function M.define_org_headline_colors(faces)
 end
 
 function M.define_highlights()
+  M.link_ts_highlights()
```

Comment:
Lets wrap this into `config:ts_highlights_enabled()` check, there is no need to define these if TS highlighting is not enabled.